src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(number) {
+      calculator.result += number;
+    },
+    subtract(number) {
+      calculator.result -= number;
+    },
+    divide(number) {
+      calculator.result /= number;
+    },
+    multiply(number) {
+      calculator.result *= number;
```

Comment:
In all methods replace `calculator` with `this. If you would change `calculator` to another  name, this way you would need to change only one line, not each method.
```suggestion
      this.result *= number;
```

Reply:
I dont know whyI can't use 'this' directly in this methods. I needed to pass an argument 'this' before that.